ci-operator/config/openshift/installer/openshift-installer-main__okd-scos.yaml

```diff
@@ -184,17 +184,6 @@ images:
       as:
       - kas-artifacts
   to: installer-artifacts
-- build_args:
-  - name: TAGS
-    value: scos
-  dockerfile_path: images/infrastructure-providers/Dockerfile
-  from: origin_scos-4.19_base-stream9
```

Comment:
the only issue here is that using the image built for ocp means the base image is rhel, whereas for OKD the base image is centos stream. in general we do avoid using rhel base images in case we leak rhel content (which is not a concern here). but it would be ideal to see images for OKD built with an scos base

Reply:
could we instead do a change similar to https://github.com/openshift/release/pull/39563 for okd-scos as well ? and in general for all the installer jobs that have been moved out separately?

Reply:
We could move the okd terraform image build to its own variant so that it could be built with a different base image, but tbh that just seems like a waste of resources, and I don't think the amount of resources here are insignificant.

We are actively working on removing the terraform image buijlds, aiming for 4.19.

Reply:
https://github.com/openshift/release/pull/62851 . For now i have split the artifacts and the terraform provider job.  We can take out the terrform-providers job once the work is done in installer to remove it. It would be good to have this split as  I am seeing lots of timeouts and we need these images for OKD releases.